generator/templates/new/plugins/example.ex

```diff
@@ -1,11 +1,28 @@
 defmodule <%= mod_name %>.Example do
   use Butler.Plugin
+  alias Butler.Plugin.Responders, as: Responder
 
-  respond(~r/example me/) do
-    reply "This is an example response"
+  @moduledoc """
+  echoes back; demonstrates parsing basic user input
+  """
+  respond ~r/echo (.*)/, conn, [_all, say] do
+    reply conn, "#{say}"
   end
 
-  hear(~r/examples/) do
-    say "I heard some examples"
+  @usage """
+  hears ping, says pong; demonstrates basic listening and a broadcast
+  """
+  hear ~r/ping/, conn do
+    # ["test"|tests]
+    say conn, "-> pong"
+  end
+
+  @moduledoc """
+  replies back with code; demonstrates escaping code blocks
+  """
+  respond ~r/code/, conn do
+    response = "console.log('this should look like code!');"
+      |> Responder.code
```

Comment:
You don't have to include the Responders. They're included with the Plugin module.